M2/Macaulay2/m2/ringmap.m2

```diff
@@ -293,6 +384,11 @@ RingMap * RingMap := RingMap => (g,f) -> (
 	  }
      )
 
+RingMap ^ ZZ := BinaryPowerMethod
```

Comment:
what's a `BinaryPowerMethod`?

Reply:
That's an excellent question!
https://github.com/Macaulay2/M2/blob/048c777dc703365e44c3865b2c1faed698b82b3a/M2/Macaulay2/d/actors.d#L525-L534
Which I'm planning to also use in
https://github.com/Macaulay2/M2/blob/d1ad3a5b8fa4f441362722d85ba6342e6a4ab09f/M2/Macaulay2/m2/varieties.m2#L356-L366